src/sfcphys.F

```diff
@@ -497,36 +492,18 @@ subroutine sfcflux(dt,ruh,xf,rvh,pi0s,ch,cq,pi0,thv0,th0,rf0,tsk,thflux,qvflux,m
     IF(imoist.eq.1)THEN
       tem = dt*dx*dy
       ! some budget calculations:
-#ifdef _B4B06R
-      !$acc update host(qvflux,ruh,rvh,rf) 
-#else
-      !$acc data create(bud1)
+      !$omp parallel do default(shared) private(i,j)
       !$acc parallel default(present) private(i,j) reduction(+:budtmp,qbsfc) 
       !$acc loop gang
```

Comment:
Do we need to add `reduction(+:qbsfc)` here?

Reply:
Thanks for the catch.

Reply:
Done